websites/G/GenoAnime/dist/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "author": {
+    "name": "DayAndNight",
+    "id": "424072110647017482"
+  },
+  "url": "genoanime.com",
+  "description": {
+    "en": "Enjoy the our website, view any anime episodes and movies"
+  },
+  "service": "GenoAnime",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/44orW6Y.png",
+  "thumbnail": "https://i.imgur.com/lANOUyc.png",
+  "color": "#42b5f7",
+  "tags": [
+    "anime",
+    "entertainment",
+    "genoanime",
```

Comment:
Remove `genoanime` as it is not an `alternate service name`
![image](https://user-images.githubusercontent.com/34070194/113783636-f2edf100-972b-11eb-935d-c1f3e4071e7e.png)

Reply:
Completed, removed from tag